src/ixdat/plotters/sec_plotter.py

```diff
@@ -2,63 +2,54 @@
 
 import matplotlib as mpl
 
-from .base_mpl_plotter import MPLPlotter
 from .ec_plotter import ECPlotter
-from .spectrum_plotter import SpectrumSeriesPlotter
+from .spectrum_plotter import SpectrumSeriesPlotter, SpectroMeasurementPlotter
 from ..exceptions import SeriesNotFoundError
 
 
-class SECPlotter(MPLPlotter):
-    """An spectroelectrochemsitry (SEC) matplotlib plotter.
-
-    FIXME: This should make use of the code in spectrum_plotter.SpectrumSeriesPlotter
-    """
+class SECPlotter(SpectroMeasurementPlotter):
+    """An spectroelectrochemsitry (SEC) matplotlib plotter."""
```

Comment:
Wow, very consistent typo :P . It might be easier for the dictionary to just correct it to my spelling ;) 
I'll make an effort to start pronouncing it that way too.